### DISCUSSION TITLE
I tried to signup to vercel using my gmail account but the account isn't working at all.

### DISCUSSION THREAD

**Discussion by bilaltufaildev** [vercel/community#4416] [Account Issue] [answered]
### Question

I tried to signup to vercel using my gmail account but the account isn't working at all. It shows that I have an account but on logging in it doesn't let me in and shows me some kind of warning: 
`We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details.`
But when I try to signup using any of the methods either email, bitbucket, or github, it shows that my account already exists. I have tried to contact the support team of vercel but it's been three to 4 days now, and no one has responded back to me in that regard.
Kindly I need help from the community as fast as possible.
Thanks

**Comment by amyegan** [comment, marked as answer]
It sounds like you already did the right thing by opening a support case. We can't safely discuss account information in public because of security concerns. 

Tickets are processed by the order in which they are received, so it can take some time for the team to respond to all of the earlier cases that were opened before they reach yours. The team is handling an unusually high number of cases at the moment, so wait times are a little longer than normal.

For more information about response times, please visit https://vercel.com/guides/vercel-support-queue-time
